.github/workflows/github-ci.yml

```diff
@@ -1,27 +1,11 @@
 name: build/deploy opensim-python-runner
 
 on:
+  workflow_dispatch:
   push:
-    branches:
-      - "*"
-    tags-ignore:
-      - "*"
-    paths-ignore:
-      - "*.md"
-      - "*.png"
-      - "*.svg"
-      - "docs/**"
-      - ".vscode/**"
+      branches: [ master ]
   pull_request:
-    branches:
-      - "*"
-    paths-ignore:
-      - "*.md"
-      - "*.png"
-      - "*.svg"
-      - "docs/**"
-      - ".vscode/**"
-      - ".vscode-template/**"
+      branches: [ master ]
```

Comment:
Pull requests are never done in the ``master`` (``main``) branch but FROM another branch INTO master
I would use 
```yaml

on:
   push:
       branches:
            - "*"
   pull_request:
        branches:
             - "*"
```

Reply:
Also your "main" branch is called ``main`` and not ``master``

Reply:
I would also keep the ignore globs

Reply:
Thanks @pcrespov. I have updated the code as you suggested. See my latest commit below (88297fb)

Reply:
But I'm not convinced about the [CI run](https://github.com/ITISFoundation/opensim-python-runner/runs/6276462134?check_suite_focus=true). 

As far as I can see, it skipped the 2 "deploy" steps. What is the expected behavior? Should those "deploy" be triggered when I create a pull request from a fork?

Reply:
@elisabettai i think it works as you instructed in the workflow 
I can see here a condition on ``master`` https://github.com/ITISFoundation/opensim-python-runner/blob/main/.github/workflows/github-ci.yml#L78

In addition, the condition should be on ``main`` and not ``master, right?

Reply:
Hi @pcrespov, thanks for checking. We'll talk in detail tomorrow, so I don't want to bother you now.

To answer to your question: now the condition in github-ci.yml is on `main` and [CI run successfully](https://github.com/elisabettai/opensim-python-runner/runs/6289575440?check_suite_focus=true) in my fork.

I've also closed this PR, the most up-to-date one is [PR #3](https://github.com/ITISFoundation/opensim-python-runner/pull/3).